### PR TITLE
DOCSP-34987: v2.23 updates

### PR DESCRIPTION
**Pull request by ccho-mongodb**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-golang/blob/master/REVIEWING.md)

JIRA - <https://jira.mongodb.org/browse/DOCSP-34987>
Staging:

[What's New](https://preview-mongodbcchomongodb.gatsbyjs.io/csharp/DOCSP-34987-v2.23-updates/whats-new/)

[Compatibility](https://preview-mongodbcchomongodb.gatsbyjs.io/csharp/DOCSP-34987-v2.23-updates/compatibility/)

## Self-Review Checklist

- [ ] Is this free of any warnings or errors in the RST?
- [ ] Did you run a spell-check?
- [ ] Did you run a grammar-check?
- [ ] Are all the links working?
- [ ] Are the [facets and meta keywords](https://wiki.corp.mongodb.com/display/DE/Docs+Taxonomy) accurate?

### CODE REVIEW

File: source/whats-new.txt [hunk 1]
```diff
@@ -10,15 +10,37 @@ What's New
...
+.. _version-2.23:
+
+What's New in 2.23
+------------------
+
+The 2.23 driver release includes the following new features:
+
+- Improvements and bug fixes to the LINQ3 integration.
+- Added support for aggregation bitwise operators.
+- Added support for the ``$out`` operator in MongoDB Time series collections.
```
Comment (mongoKart) [inline source/whats-new.txt:40]:
```suggestion
- Added support for the ``$out`` operator in MongoDB time series collections.
```

File: snooty.toml [hunk 2]
```diff
@@ -26,11 +26,11 @@ language = "C#"
 mongo-community = "MongoDB Community Edition"
 mongo-enterprise = "MongoDB Enterprise Edition"
 docs-branch = "master"                                                                                          # always set this to the docs branch (i.e. master, 1.7, 1.8, etc.)
-version-number = "2.22"                                                                                         # always set this to the driver branch (i.e. 1.7.0, 1.8.0, etc.)
+version-number = "2.23"                                                                                         # always set this to the driver branch (i.e. 1.7.0, 1.8.0, etc.)
 version = "v{+version-number+}"
 example = "https://raw.githubusercontent.com/mongodb/docs-csharp/{+docs-branch+}/source/includes/code-examples"
 stable-api = "Stable API"
-api-root = "https://mongodb.github.io/mongo-csharp-driver/{+version-number+}/html"
+api-root = "https://mongodb.github.io/mongo-csharp-driver/2.23.0/api/"
```
Comment (mongoKart) [inline snooty.toml:33]: FYI: My PR will overwrite this change

**Review by mongoKart — APPROVED** [review]
LGTM w/ a suggestion and a note

File: snooty.toml (hunk 2 above)
Comment (ccho-mongodb) [reply snooty.toml:33]: Ok, reverting. I will handle any merge conflicts.

**Pull request merged**